openlibrary/templates/books/edit/edition.html

```diff
@@ -46,27 +47,32 @@
     </div>
 
 $jsdef render_work_autocomplete_item(item):
-    <div class="ac_work" title="Select this work">
-        <div class="cover">
-            $if item.cover_i:
-                <img src="https://covers.openlibrary.org/b/id/$(item.cover_i)-M.jpg" alt="Cover of $item.title">
+    $if item.key == "__new__":
+        <div class="ac_work ac_addnew">
+            <span class="action">$_('>> Move to a new work (copies work data from current work)')</span>
```

Comment:
for strings like this I'd move the `>>` outside the localisation fn. Probably a minor comment, but well do for rememberinf to wrap the new strings in `$_()` in the first place! :)

We should have some guidelines for what sort of things to put inside and outside

Reply:
I'm actually not sure which is best; I was wondering that as well. My guess would be this matters for e.g. RTL languages? So I figured I'd just leave it in the hopes translators will handle it correctly; marking as won't fix.

Reply:
I think generally we should keep text and markup separate.  The  ">>" feels like ad hoc textual markup to me. If we can't eliminate it, it should stay in the translatable portion so that it can be rearranged by the translators.